CHANGELOG.md

```diff
@@ -3706,7 +3706,7 @@ Anyway, the cluster should get back to the proper size after 10 min.
     clusters post upgrade where kubelet hard eviction has been turned on for
     memory. To opt-out set `--experimental-allocatable-ignore-eviction=true`.
   * More details on these feature here:
-    https://kubernetes.io/docs/concepts/cluster-administration/node-allocatable/
+    https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/
```

Comment:
Thankyou for you review
Well, in my mind,the changelog should always be useful when some people need to compare difference between versions,so the link should also be useful for people understanding the change,how do you think?